lib/good_job.rb

```diff
@@ -290,7 +290,8 @@ def self.deprecator
   # For use in tests/CI to validate GoodJob is up-to-date.
   # @return [Boolean]
   def self.migrated?
-    GoodJob::Job.concurrency_key_created_at_index_migrated?
+    GoodJob::Job.concurrency_key_created_at_index_migrated? &&
+      GoodJob::Job.job_class_index_migrated?
```

Comment:
You can just check for the latest migration, no need to check for any of the past ones

```suggestion
    GoodJob::Job.job_class_index_migrated?
```